src/auxil/auxil/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gfx-auxil"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
this should probably be `0.6.0`, since spirv-cross is the public dependency

Reply:
Good point – we'll need this change on `master` too then